docs/en/latest/plugins/aws-lambda.md

```diff
@@ -0,0 +1,156 @@
+
+### IAM Authorization Schema
+
+| Name             | Type   | Requirement  | Default        | Valid       | Description                                                                                |
+| -----------      | ------ | -----------  | -------        | -----       | ------------------------------------------------------------                               |
+| accesskey        | string | required     |                |             | Genereated  access key ID from aws IAM console.                                            |
+| secret_key       | string | required     |                |             | Genereated access key secret from aws IAM console.                                         |
+| aws_region       | string | optional     | "us-east-1"    |             | The aws region where the request is being sent.                                            |
+| service          | string | optional     | "execute-api"  |             | The service that is receiving the request (In case of Http Trigger it is "execute-api").   |
+
+## How To Enable
+
+The following is an example of how to enable the aws-lambda faas plugin for a specific route URI. Calling the apisix route uri will make an invocation to the lambda function uri (the new upstream).  We are assuming your cloud function is already up and running.
+
+```shell
+# enable aws lambda for a route via api key authorization
+curl http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+{
+    "plugins": {
+        "aws-lambda": {
+            "function_uri": "https://x9w6z07gb9.execute-api.us-east-1.amazonaws.com/default/test-apisix",
+            "authorization": {
+                "apikey": "<Generated API Key from aws console>",
+            },
+            "ssl_verify":false
+        }
+    },
+    "uri": "/aws"
+}'
+```
+
+Now any requests (HTTP/1.1, HTTPS, HTTP2) to URI `/aws` will trigger an HTTP invocation to the aforesaid function URI and response body along with the response headers and response code will be proxied back to the client. For example ( here aws lambda function just take the `name` query param and returns `Hello $name` ) :
+
+```shell
+$ curl -i -XGET localhost:9080/aws\?name=apisix
+HTTP/1.1 200 OK
+Content-Type: application/json
+Connection: keep-alive
+Date: Sat, 27 Nov 2021 13:08:27 GMT
+x-amz-apigw-id: JdwXuEVxIAMFtKw=
+x-amzn-RequestId: 471289ab-d3b7-4819-9e1a-cb59cac611e0
+Content-Length: 16
+X-Amzn-Trace-Id: Root=1-61a22dca-600c552d1c05fec747fd6db0;Sampled=0
+Server: APISIX/2.10.2
+
+"Hello, apisix!"
+```
+
+For requests where the mode of communication between the client and the Apache APISIX gateway is HTTP/2, the example looks like ( make sure you are running APISIX agent with `enable_http2: true` for a port in conf.yaml or uncomment port 9081 of `node_listen` field inside [config-default.yaml](../../../../conf/config-default.yaml) ) :
```

Comment:
Better to use `config-default.yaml` directly, since people may read the doc on the doc's website. Please fix the azure-functions' doc too.

Reply:
Updated